app/src/test/java/space/taran/arkmemo/SVGUtilsTest.kt

```diff
@@ -0,0 +1,27 @@
+package space.taran.arkmemo
+
+import junit.framework.TestCase.assertEquals
+import org.junit.Test
+import space.taran.arkmemo.utils.SVG
+
+class SVGUtilsTest {
+
+    @Test
+    fun parses_correctly() {
+        val svg = SVG.parse(svgString)
+        assertEquals(svg.get(), svgString)
```

Comment:
Let's rename `get` as `generate`. We should make phases of parsing and generating explicit.